format.py

```diff
@@ -43,8 +46,25 @@ def yaml_sort(d):
     return d
 
 
+# this still doesn't deal with nested dictionaries in lists
+# ex [{a: b}] -> [a: b]
+def yaml_sort_flow(d):
+    if isinstance(d, dict):
+        result = ruamel.yaml.CommentedMap()
+        result.fa.set_flow_style()
+        for k in d.keys():
+            result[k] = yaml_sort_flow(d[k])
+        return result
+    elif isinstance(d, list):
+        for i, item in enumerate(d):
+            d[i] = yaml_sort_flow(item)
+
+    return d
+
 def main():
     yaml = ruamel.yaml.YAML()
+    yaml.preserve_quotes = True
+    # yaml.default_flow_style = None
```

Comment:
```suggestion
```

Reply:
Currently, I've been manually making sure the three copies produce the same output, but most of the tests should have the same WDL code with only the version number being the difference. I think most tests should not be too different between different versions, so I can try generating WDL tests programatically first.